Tests/DependencyInjection/AbstractDoctrineExtensionTest.php

```diff
@@ -1281,7 +1396,6 @@ private function loadContainer(
     ): ContainerBuilder {
         $container = $this->getContainer($bundles);
         $container->registerExtension(new DoctrineExtension());
-        $container->addCompilerPass(new CacheCompatibilityPass());
```

Comment:
This seems contraproductive. It means we still don't have a test which checks our cache compatibility layer works with second level cache.

Reply:
I see now that it was removed because it's already included in `compileContainer` function